### PR TITLE
Add a section blog post feature to the project

### PR DESCRIPTION
**Pull request by bernyhas**
Add a feature section blog to the website any suggestion are welcome

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -34,10 +38,10 @@
   header {
     margin-bottom: 95px;
```
Comment (bernyhas) [inline style.css:39]: Hello @bernyhas ,
You do pretty well but unfortunately we request you to improve your css styles.

Happy coding

**Review by bernyhas — COMMENTED** [review]
Hi @bernyhas 

Please working on your css style improve if you have any question please let us know!
Cheers

**Review by Lameck1 — COMMENTED** [review]
Hi @bernyhas , 

While you made a great effort in this project, this code review request will unfortunately not commence.
You've done well implementing some features that were asked in the project requirements, however, I will mark this project as invalid. Read the comments below to understand why.

#### :warning: Please work on the following: 👇🏼 :

- [ ] Consider reorganizing your work. Things kind of look mixed up. There is work already merged into the main branch bearing in mind that this is the first task on review.
- [ ] Please make the PR title descriptive as per the current task being submitted for review.
- [ ] The branch being submitted for review must have a professionally documented README. Please work on this.

That's it for now.
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by bernyhas** [comment]
As per code reviewer @Lameck1  requested to validate the project Update the PR Title as descriptive possible and also the readme to the feature branch and well documented as per github flow guidlines

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @bernyhas , 

Great job so far as following the design guide however, there are some issues that you still need to work on to go to the next project but you are almost there!


# Required Changes ♻️

- [ ] The logo at the top left which has your name in this case is missing the right semantic tag, please use an `a`  tag 
- [ ] Please use a `section` tag for headline section, I can see you made use of a `div` tag
- [ ] The headline title is a `h1` tag,  Kindly update it
- [ ] The list of icons in the headline sections needs to be wrapped in a `ul` tag and an `li` for each icon. Kindly update it
- [ ] please add a line break after your name in the headline title so the title reads `I'm Hans` instead of `I'm Hans Glad`

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request closed without merging**